user-guide/gpu.rst

```diff
@@ -164,53 +166,61 @@ could look like:
 
 ::
 
-   #!/bin/bash
-   #
-   #PBS -N cuda_test
-   #PBS -q gpu
-   #PBS -l select=1:ncpus=40:ngpus=4
-   #PBS -l walltime=0:20:0
-   # Budget: change 't01' to your budget code
-   #PBS -A t01
-
-   # Load the required modules (this assumes you compiled with GCC 6.3.0)
-   module load cuda
-   module load gcc/6.3.0
+    #!/bin/bash
+    #
+    # Slurm job options (name, compute nodes, job time)
+    #SBATCH --job-name=CUDA_Example
+    #SBATCH --time=0:20:0
+    #SBATCH --partition=gpu-skylake
+    #SBATCH --qos=gpu
+    #SBATCH --exclusive
+    #SBATCH --gres=gpu:4
+    #SBATCH --nodes=1
+    #SBATCH --tasks-per-node=1
+    #SBATCH --cpus-per-task=40
 
-   cd $PBS_O_WORKDIR
+    # Replace [budget code] below with your project code (e.g. t01)
+    #SBATCH --account=[budget code]
+    
 
-   ./cuda_test.x
+    # Load the required modules 
+    module load nvidia/cuda-11.0
 
-The line ``#PBS -l select=1:ncpus=40:ngpus=4`` requests 1 node, 40 cores on that node and 4 GPU
-accelerators on that node (i.e. a full GPU compute node).
 
-Job submission script using multiple GPUs on multiple nodes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    srun ./cuda_test.x
 
-.. note:: Remember that there are a maximum of 4 GPU accelerators per node and a maximum of 40 CPU cores per node.
+..
+   The line ``#PBS -l select=1:ncpus=40:ngpus=4`` requests 1 node, 40 cores on that node and 4 GPU
+   accelerators on that node (i.e. a full GPU compute node).
 
-A job script that required 8 GPU accelerators and 80 CPU cores for 20 minutes across 2 nodes
-could look like:
+..
+   Job submission script using multiple GPUs on multiple nodes
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-::
+   .. note:: Remember that there are a maximum of 4 GPU accelerators per node and a maximum of 40 CPU cores per node.
 
-   #!/bin/bash
-   #
-   #PBS -N cuda_test
-   #PBS -q gpu
-   #PBS -l select=2:ncpus=40:ngpus=4
-   #PBS -l walltime=0:20:0
-   # Budget: change 't01' to your budget code
-   #PBS -A t01
-
-   # Load the required modules (this assumes you compiled with GCC 6.3.0)
-   module load cuda
-   module load gcc/6.3.0
-   module load mpt
+   A job script that required 8 GPU accelerators and 80 CPU cores for 20 minutes across 2 nodes
+   could look like:
+
+   ::
+
+      #!/bin/bash
+      #
+      #PBS -N cuda_test
+      #PBS -q gpu
+      #PBS -l select=2:ncpus=40:ngpus=4
+      #PBS -l walltime=0:20:0
+      # Budget: change 't01' to your budget code
+      #PBS -A t01
+
+      # Load the required modules (this assumes you compiled with GCC 6.3.0)
+      module load cuda
+      module load gcc/6.3.0
+      module load mpt
```

Comment:
This still seems to be PBS?

Reply:
yes, it was commented out until we find more about nnodes,ngpus,ncpus,etc